includes/Admin/Admin_Page.php

```diff
@@ -22,6 +22,14 @@ class Admin_Page {
 	 */
 	protected $admin_ajax;
 
+	/**
+	 * Admin page hook suffix.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	protected $hook_suffix;
```

Comment:
We should initialize this as an empty string, otherwise the `@var string` above is incorrect, as it could also be `null` then.
```suggestion
	protected $hook_suffix = '';
```